src/CMakeLists.txt

```diff
@@ -28,6 +28,13 @@ include(ROCMInstallTargets)
 include(ROCMPackageConfigHelpers)
 include(RegisterOp)
 include(CheckCXXLinkerFlag)
+ 
+function(ReadSHA InputFile VAR)
+    file(STRINGS "${InputFile}" String_output)
+    add_compile_definitions(${VAR}="${String_output}")
+endfunction()
```

Comment:
I don't think this needs to be a separate function, as its only used in one place and is only two lines.

Reply:
Moved to below and removed the function wrapping